module/platform/OpenCR/HardwareIO/src/HardwareIO.cpp

```diff
@@ -284,7 +211,10 @@ namespace module::platform::OpenCR {
                         // Start the packet watchdog since the main loop is now starting
                         model_watchdog.disable();
                         model_watchdog.unbind();
-                        log<NUClear::WARN>("Packet watchdog enabled");
+
+                        log<NUClear::INFO>("Packet watchdog enabled");
+
+                        packet_watchdog = create_packet_watchdog();
                         packet_watchdog.enable();
```

Comment:
Just for my own sake, this should be the only spot where the `packet_watchdog` is first enabled, right? Is it disabled before?

Reply:
correct, it isn't enabled anywhere else.

This did make me realise the `.enable()` does nothing, so I'll remove it for both the packet and model watchdogs.